### PR TITLE
wp-load.php path correction

### PR DESCRIPTION
**Pull request by sioannides**

### CODE REVIEW

File: includes/ajax/wp-sms-subscribe.php [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?php
-	include_once("../../../../../wp-load.php");
+	include_once("../../../../../wp/wp-load.php");
```
Comment (mamchenkov) [inline includes/ajax/wp-sms-subscribe.php]: @sioannides this will break for non wp/ folder installed WordPress sites.  Need a better solution.

**Comment by sioannides** [comment]
@mamchenkov Can you please merge this, I need it for the http://test1.themoc.qobocloud.com

**Comment by mamchenkov** [comment]
@sioannides I still don't like this solution.  It looks ugly and feels fragile.  Have you looked into WordPress Codex for the recommended way of doing this?

**Comment by mamchenkov** [comment]
@sioannides here is the [general direction](https://codex.wordpress.org/AJAX_in_Plugins)

**Comment by sioannides** [comment]
@mamchenkov You realize that this will take time...I'll have to change many things on the plugin to get it working like that.

**Comment by mamchenkov** [comment]
@sioannides yes, I do.  But it's not THAT much.  The initial plugin wasn't written according to WordPress recommendations, and it broke for us.  We need to fix it to use it.  If we fixed it like you are proposing, we'll have to maintain this forever.  Instead of doing a quick hack of a fix, I suggest we fix it properly.  If we do, we can send the patch to the plugin maintainers, and hopefully they'll merge it in, and we won't have to maintain our own version.

I'm not asking to fix all the issues in the plugin.   Just this specific one.

On top of it, this issue is actually quite common with other plugins.  We now maintain a separate version of the Landing Pages plugin ( @constantinoschristoforou did the change there ), with very similar troubles.  Even though the original authors asked us to send the patch.  (We said we gonna do that later, cause we need to clean it up, but we never did ).

If we do quick fixes here and there, then we end up with tonnes of these lose ends.  We need to maintain them.  They get outdated.  They are polluting our repository space.  And we nobody is benefiting.  If we send patches upstream, we are the first to benefit.  But then there is also the rest of the world too.  We get karma, among other things. :)

And, on top of that, for once, I'm not gonna be the only one complaining about Pull Requests. :imp:

**Comment by sioannides** [comment]
@mamchenkov I changed the plugin

**Comment by mamchenkov** [comment]
@sioannides beautiful! :+1:   We'll need to send this patch upstream.

**Pull request merged**